app/models/job.rb

```diff
@@ -208,4 +208,12 @@ def set
     @set = SetClient::Set.find(set_uuid).first
   end
 
+  def work_plan
+    work_order.work_plan
+  end
+
+  def work_plan_id
```

Comment:
Is this just used for the JSONAPI resource / do we need it

Reply:
Yep and yep